packages/browser/src/login/oidc/redirectHandler/AggregateRedirectHandler.ts

```diff
@@ -39,7 +39,7 @@ import {
  */
 @injectable()
 export default class AggregateRedirectHandler
-  extends AggregateHandler<[string], ISessionInfo | undefined>
+  extends AggregateHandler<[string], ISessionInfo & { accessToken: string }>
```

Comment:
Is this an interim step, or is it intended that the `accessToken` always be passed about like this? (I was thinking instead we'd be able to push the ever-so-precious accessToken directly into the fetch closure **_immediately_** upon getting it from `oidc-client-js`, and never need to pass it around anywhere (not that I've thought through if that's feasible or not :) !?)

Reply:
We could push the call to `buildBearerFetch` one level down and return the `fetch` instead of the `accessToken`, but as it is right now the token is ever only accessible in the `handleRedirect` scope, from which it is stored in the `fetch` scope, so I don't think moving the call would make a difference. I find it slightly clearer that the fetch is built by the object that will then expose it, what do you think ?

Reply:
Yeah, is certainly debatable. My intuition says move it down one level - I'm just a tad uncomfortable seeing exposure of the 'concept' of an `accessToken` across all the redirect handlers. I'm not saying that's wrong, but it just feels a tiny bit safer if redirect handlers all return fetch functions instead of super-secret accessTokens (not that them returning fetch functions is super-intuitive either of course). So unless you've any further misgivings about doing it, I'd move it down.